### DISCUSSION TITLE
3-scale tesselation from text files

### DISCUSSION THREAD

**Discussion by qdolle15** [neperfepx/neper#363] [Q&A] [answered]
Hello,

I would like to create a 3-scale tessellation with my own coordinates for each cells and each seeds. But I don't understand how to deal with my text files or how should I shape its ? And what is the command line associated ? 

I already look at the appendix B.3 of the Neper doc but I perhaps misunderstood something.

Is-it possible to have an example ? Thanks

**Comment by rquey** [comment, marked as answer]
See #401 for a related example.
